portfolio/src/main/webapp/script.js

```diff
@@ -31,27 +37,52 @@ function addRandomQuote() {
   quotesContainer.innerText = quote;
 }
 
-// Written with help from https://www.w3schools.com/howto/howto_html_include.asp
+/**
+ * Generates a URL for a random image in the images directory and adds an img
+ * element with that URL to the page.
+ */
+function randomizeImage() {
+  const imageIndex = Math.floor(Math.random() * 18) + 1;
+  const imgUrl = 'images/cats/cats-' + imageIndex + '.jpg';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
+  imgElement.setAttribute('height', '500');
+
+  const imageContainer = document.getElementById('cats-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
```

Comment:
Maybe put a placeholder cat image in html, such as cat-1 so that you don't need this hacky way to remove and re-add the image?